src/Sulu/Bundle/MarkupBundle/Markup/MarkupParser.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MarkupBundle\Markup;
+
+use Sulu\Bundle\MarkupBundle\Tag\TagRegistryInterface;
+
+/**
+ * Parses html content and replaces special tags.
+ */
+class MarkupParser implements MarkupParserInterface
```

Comment:
Should we maybe call that `HtmlMarkupParser`, in case we also add something similar for e.g. XML or RSS in the future?